dashboard/config/scripts_json/csd3-2023.script_json

```diff
@@ -20601,7 +20601,7 @@
       "key": "57428b8b-954f-4e25-b6cf-1dd6338ef623",
       "position": 3,
       "learning_goal": "Program Development - Program Sequence",
-      "ai_enabled": true,
+      "ai_enabled": false,
```

Comment:
this represents another config bug caught by `validate_learning_goals`. This learning goal does not have a corresponding entry in standard_rubric.csv.